client/styles/layout/_fullscreen.scss

```diff
@@ -5,13 +5,33 @@
   flex-flow: column;
 }
 
+.fullscreen-preview__header {
+  text-align: center;
+  font-size: 16px;
+  font-family: Montserrat, sans-serif;
+  margin-bottom: 5px;
+}
+
 .fullscreen-preview__title {
   width: 100%;
+  color: #ed225d;
```

Comment:
rather than adding these colors inline, you'll need to pull them from variables in `client/styles/abstracts/_variables.scss`. this should also pull from whatever the user has set as their theme! and by user i mean person logged into the editor, rather than the owner of the sketch.

Reply:
I'm currently trying to resolve this, but I'm kind of having a problem with sass : the `@include` that I'm adding to the `_fullscreen.scss` file are not taken into account in the developpement environment (whereas if I add them to another file, it works perfectly). Has anyone ever had a similar problem ?